examples/dojo_simple/src/lib.cairo

```diff
@@ -68,10 +85,41 @@ pub mod actions {
             let caller = starknet::get_caller_address();
 
             let position = Position { player: caller, x: 1, y: 2 };
-            position.set(world);
 
+            world.set(@position);
+            let _position: Position = world.get(caller);
             emit!(world, PositionUpdated { player: caller, new_x: 1, new_y: 2 });
         }
+
+        fn despawn(ref world: IWorldDispatcher) {
+            let caller = starknet::get_caller_address();
+            ModelStore::<Position>::delete_from_key(world, caller);
+        }
+
+        fn move(ref world: IWorldDispatcher, new_x: u32, new_y: u32) {
+            let caller = starknet::get_caller_address();
+            let mut position: Position = world.get(caller);
+            position.x = new_x;
+            position.y = new_y;
+            world.set(@position);
+            emit!(world, PositionUpdated { player: caller, new_x, new_y });
+        }
+
+        fn get_position(world: @IWorldDispatcher) -> Point {
+            let caller = starknet::get_caller_address();
+            let entity: PositionEntity = world.get_entity(caller);
```

Comment:
If we can do this:
```rust
    let mut position: Position = world.get(caller);
    position.x = new_x;
    position.y = new_y;
    world.set(@position);
```
I'm wondering if we still really need this concept of `Entity` 🤔

Reply:
Have a look at this